includes/discovery/vlans/q-bridge-mib.inc.php

```diff
@@ -19,7 +22,39 @@
             echo '+';
         }
 
-        $device['vlans'][$vtpdomain_id][$vlan_id] = $vlan_id;
+        $key = "0.$vlan_id";
+
+        $untagged_port_indices = q_bridge_bits2indices($untag[$key]['dot1qVlanCurrentUntaggedPorts']);
+        $tagoruntag_indices = q_bridge_bits2indices($tagoruntag[$key]['dot1qVlanCurrentEgressPorts']);
+        $tagged_port_indices = array_diff($tagoruntag_indices, $untagged_port_indices);
+
+        foreach ($tagged_port_indices as $port_ifindex) {
+            $port = get_port_by_index_cache($device, $port_ifindex);
+            echo str_pad("q-bridge ", 10).str_pad($port_ifindex, 10).str_pad($port['ifDescr'], 25).str_pad("n/a", 10).str_pad("n/a", 15).str_pad("n/a", 10);
```

Comment:
Unless you need to you should use d_echo() to output things. echo() is fine when need to let the user know the poller module is doing but this should be short, brief and a one liner.